### ISSUE TITLE
Research setting aspect-ratio

### ISSUE THREAD

**Issue by Witoso** [ckeditor/ckeditor5#14366]
After #14313 we want to check if we should set on the model, and later on inline styles in the data output an `aspect-ratio` style.

This attribute could help us to:
    1. remove `height:auto` style from the `img`.
    2. show non-proportionally resized images that come from CKEditor 4 for example.
   
Outcome:   
Research setting inline `aspect-ratio` based on `resizedHeight & resizedWidth` or fallback to attributes `width` and `height`.

**Comment by Witoso** [comment]
Closing as duplicate of #14357.
